Voycar.Api.Web/Generic/Endpoint/Delete/Single.cs

```diff
@@ -19,6 +19,21 @@ public override void Configure()
     {
         this.Delete(typeof(TEntity).Name.ToLowerInvariant() + "/{id}");
         this.Roles(this.roles);
+        Description(b => b
+                .Accepts<Entity>("Voycar.Api.Web/Generic/Entity")
+                .Produces<IResult>(200)
+                .ProducesProblem(404),
+            clearDefaults: true);
+        Summary(s =>
+        {
+            s.Summary = $"DELETE Endpoint for {typeof(TEntity).Name}";
+            s.Description = $"This Endpoint is used to remove {typeof(TEntity).Name} objects from the database";
+            s.Responses[200] = "OK response if DELETE operation was successful";
+            s.Responses[404] =
+                "Not Found response if DELETE operation was performed for on an Entity that could not be found in the database";
```

Comment:
Hier auch wieder auf die formulierung achten, "for on an" existiert so nicht. Entweder das eine oder das andere.